docs/uptime/analytics/open-web-analytics.md

```diff
@@ -0,0 +1,150 @@
+keywords: ‘open, web, analytics,’
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 
+modified: 
+modified_by:
+    name: 
+title: 'Open Web Analytics (OWA) on CentOS 6.5 and 7'
+contributor:
+    name: Douglas Colby
+external_resources:
+ - '[OWA Website](http://www.openwebanalytics.com)'
+ - '[OWA Forum](http://www.openwebanalytics.com/?page_id=4)'
+---
+
+##How to Install Open Web Analytics on CentOS 6.5 and 7 (with MySQL back-end)
+
+Open Web Analytics (OWA) is an open source alternative to commerical web analytics software. Use it to track and analyze traffic on your websites and applications. OWA analytics can easily be added to pages with simple Javascript, PHP, or REST based APIs. OWA also comes with built-in support for tracking websites made with popular content management frameworks such as WordPress and MediaWiki.
+
+This guide assumes that you have your Linode already set up and running, that you have followed the [Getting Started](/docs/getting-started) and [Securing Your Server](/docs/security/securing-your-server) guides, that the Linode's [hostname is set](/docs/getting-started#setting-the-hostname), and that you are familiar with the concept of the command line interface (CLI). Since this guide concerns the installation and configuration of software packages all commands are to be run as the root user.
+
+##1. Install supporting software
+
+1. Install the supporting software packages by running the following command:
+
+    yum install httpd php php-mysql mysql mariadb-server
+
+{: .note}
+>
+>`yum` will not reinstall packages that already exist on the system. Therefore this command is safe to run even if you have some of the software installed already. 
+
+2. Always a good idea to make sure your system is up-to-date:
+
+    yum update
+
+3. CentOS 7 users run the following commands (if you are on CentOS 6.5 skip to the next command):
+
+    systemctl enable mariadb
+    systemctl enable httpd
+
+4. CentOS 6.5 users run:
+
+    chkconfig mysql on
+    chkconfig httpd on
+
+##2. Set up MySQL
+
+2. Then (if you already have mysql installed you can skip this step):
+
+    mysql_secure_installation
+
+Answer yes to all questions -- make sure to set a strong password for root (press enter when prompted for current root password since it is initially blank). This is the root user for mysql and is not related the systems root user.
+
+3. Next run the following. Enter your mysql root password when asked:
+ 
+    mysql -u root -p
+
+4. The following SQL commands will create a database named `owadb` with a user named `owadbuser` and with the password `owadbpassword`. Be sure to change the password to something better of your own choice. This information will be needed later to configure OWA.
+
+    CREATE DATABASE owadb;
+    GRANT ALL PRIVILEGES ON owadb.* TO owadbuser@localhost IDENTIFIED BY 'owadbpassword';
+    FLUSH PRIVILEGES;
+    quit
+
+5. Now you need to restart your Linode:
```

Comment:
Why would you need to reboot your Linode at this point? If it's to start the services, that won't matter because:
a) you need to start the mariadb service before you can do any of this section, and 
b) you can start both services from the command line.